Failed CI/Syntax

See https://github.com/gravitystorm/openstreetmap-carto/actions/runs/99555421

 ```
Set up xmllint 4s
   Run sudo apt-get install -qq --no-install-recommends libxml2-utils
   E: Failed to fetch http://ppa.launchpad.net/ondrej/php/ubuntu/pool/main/libx/libxml2/libxml2-    utils_2.9.10+dfsg-2+ubuntu18.04.1+deb.sury.org+1_amd64.deb  404  Not Found [IP: 91.189.95.83 80]
   E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
##[error]Process completed with exit code 100.
```

This did not affect the rendering, and I don't see any changes in the code that could have caused this. Is there some problem with the linked server at ppa.launchpad.net perhaps?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -13,6 +13,8 @@ jobs:
     - name: Set up Python
       run: |
         python -m pip install pyyaml colormath
+    - name: Set up apt
+      run: sudo apt-get update -qq
     - name: Set up xmllint
       run: sudo apt-get install -qq --no-install-recommends libxml2-utils
     - name: Set up shell

